### PR TITLE
fix Volcano

### PR DESCRIPTION
**Pull request by ksandronline**
in conf file add the original creators credit.
in mts file fix:
- replace all ctf_map:dirt_with_dry_grass to default:dirt_with_dry_grass
- replace all ctf_map:sand to default:sand
+ add more sand
(and add more sand_with_kelp but his not have in CTF game :( )

### CODE REVIEW

**Comment by ksandronline** [comment]
screenshot_20210701_125217

**Comment by ksandronline** [comment]
or you can cancel this commit (I do not know if this is possible) https://github.com/MT-CTF/maps/pull/226/commits/7e9709fdfd69aa2ab036a1a702353765e7c74ed5

File: volcano/map.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 license = CC BY-SA 4.0
+others = To create the airships I used a schematic from AAmdahl (https://www.planetminecraft.com/project/chateau-de-lufthausen---flying-mansion/)
```
Comment (Panquesito7) [inline volcano/map.conf:2]:
```suggestion
others = To create the airships, I used a schematic from AAmdahl (https://www.planetminecraft.com/project/chateau-de-lufthausen---flying-mansion/)
```

**Review by MoreKat — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Didn't test, but the changes in `map.conf` look good. 👍

**Pull request merged**